prime-router/build.gradle.kts

```diff
@@ -34,7 +34,7 @@ plugins {
     id("org.flywaydb.flyway") version "9.21.2"
     id("nu.studer.jooq") version "7.1.1"
     id("com.github.johnrengelman.shadow") version "7.1.2"
-    id("com.microsoft.azure.azurefunctions") version "1.13.0"
```

Comment:
This seems to be reverting a recently merged dependabot update. We should probably leave this alone.